java/sample/config/properties.json

```diff
@@ -0,0 +1,28 @@
+{
+  "piranhaMethodProperties": [
+    {
+      "methodName": "flagDisabled",
```

Comment:
how is the `flagDisabled("some_flag")` differentiated from `flagDisabled()`? Even if argument indices are not handled in this PR, I think it is important to handle the differential when  the flagname is one of the arguments and when it is not.

Reply:
I've added a check under `if (mit.getArguments().size()==0)` (it comes around here: https://github.com/uber/piranha/blob/dd050f86315ab3e719fa5d22d7f0039ad844b039/java/piranha/src/main/java/com/uber/piranha/XPFlagCleaner.java#L289) 

that does this:
```
if (mit.getArguments().isEmpty()) {
          if (xpFlagName.equals(EMPTY)) {
            return getXPAPIFromMIT(mit);
          } else {
            return API.UNKNOWN;
          }
   }
```

So a user would have to explicitly run Piranha twice (with different flag arguments) to delete both`flagDisabled("some_flag")` and `flagDisabled()`

I've updated the relevant testcase as well.

Reply:
This will be problematic. Why not simply add an additional argument index to the json file, and if the argument index is not provided, flagname need not be checked. Otherwise, for the given index, the flag name is checked.

Reply:
Added argumentIndex to properties.json and discarded this temporary fix.